transform/mattermost-analytics/models/marts/product/nps/_nps__models.yml

```diff
@@ -37,3 +37,94 @@ models:
       - name: user_email
         description: The email of the user who submitted feedback (optional).
         tags: [pii]
+
+  - name: fct_nps_score
+    tests:
+      - dbt_utils.expression_is_true:
```

Comment:
nit: Tests can also have a name so that DBT output is a bit more readable. Here's how a name can be added: https://docs.getdbt.com/reference/resource-properties/data-tests#custom-data-test-name

Reply:
Added names to the tests 
`count_nps_users_check_daily`
`count_nps_users_check_last90d`